action/MoveGroupSequence.action

```diff
@@ -0,0 +1,13 @@
+# A list of motion commands
```

Comment:
As this is a high level action, can you add a bit more comments at the top of the file describing the use case of these messages? Aka inline documentation please :-)

Reply:
please see my latest commit